### PR TITLE
[FX-4586] Remove Popover

### PR DESCRIPTION
**Pull request by TomasSlama**
[FX-4586]

### Description

As Popover is not used anywhere we can remove it.

Tested in Client Portal https://github.com/toptal/client-portal/pull/8597
Tested in Staff Portal https://github.com/toptal/staff-portal/pull/12450

### How to test

- [Temploy](https://picasso.toptal.net/fx-4586-popover)
- FIXME: Add the steps describing how to verify your changes

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- Make sure that additions and changes on the design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- Annotate all `props` in component with documentation
- Create `examples` for component
- Ensure that deployed demo has expected results and good examples
- Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- Covered with tests ([visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md) included)

**Breaking change**

- codemod is created and showcased in the changeset
- [x] test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-4586]: https://toptal-core.atlassian.net/browse/FX-4586?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by TomasSlama** [comment]
@toptal-bot run package:alpha-release

File: packages/base/Dropdown/src/index.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
 export * from './DropdownCompound'
 export * from './Dropdown'
 export * from './DropdownArrow'
-export * from './Popover'
```
Comment (sashuk) [inline packages/base/Dropdown/src/index.ts:4]: Should the newline be kept?

File: .changeset/nine-countries-shake.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@toptal/picasso-dropdown': major
+---
+
+- remove unused Popover
```
Comment (sashuk) [inline .changeset/nine-countries-shake.md:5]: NIT: this is the most "NIT" comment possible, but [maybe we could use ](https://english.stackexchange.com/questions/52508/difference-between-delete-and-remove)`delete` here 🙂

**Review by sashuk — APPROVED** [review]
Code looks good, as well as pull requests and temploys where it was tested!

File: packages/base/Dropdown/src/index.ts (hunk 1 above)
Comment (TomasSlama) [reply packages/base/Dropdown/src/index.ts:4]: yes, I think we have eslint rule for that 🤔

**Comment by TomasSlama** [comment]
@toptal-anvil ping reviewers

**Review by mkrl — APPROVED** [review]
Looks good, thanks! Happy to see something unused gone :)

**Pull request merged**